apps/zk-explorer/app/page.tsx

```diff
@@ -1,34 +1,235 @@
-import { Card, Typography } from '@webb-tools/webb-ui-components';
-import { ArrowRightUp } from '@webb-tools/icons';
-import { ProjectSubmissionControls } from '../components/ProjectSubmissionControls';
+'use client';
```

Comment:
find a way of not using 'use client' on page.tsx file

Reply:
@yurixander After searching around the Next.js document, one way I found is that you can utilize the [`useSearchParams`](https://nextjs.org/docs/app/api-reference/functions/use-search-params) to manage the search/filter state of the home page. You can try to move the hooks to smaller components with this, making each component client-side without any need to wrap those into a new component. Hope this helps!

Reply:
Great find. I'll move to implement those changes. You're right, we should definitely SSR the landing page for best SEO.